test/utils.test.ts

```diff
@@ -1,111 +0,0 @@
-import { BigNumber, parseFixed } from '@ethersproject/bignumber';
-import { expect } from 'chai';
```

Comment:
we should still somehow test that we are scaling correctly

Reply:
Similar to: https://github.com/re-nft/sdk/pull/10#discussion_r1121596488

If we remove the SDK's nested assumptions about scaling for Whoopi, these operations disappear and it the logic is decided by the implementor.